### DISCUSSION TITLE
✅ Vibration toggle

### DISCUSSION THREAD

**Discussion by mscuttari** [isontheline/pro.webssh.net#825] [Ideas]
Can you please add a toggle button to disable vibration? I would like to disable it globally, but it would be cool also for it to be controllable on a per-action basis (tab, erasing when line is empty, etc). I personally find myself using the Tab button a lot and getting the device vibrating every time is a bit annoying.

Thanks

**Comment by isontheline** [comment]
Hello @mscuttari 👋

Thanks for using WebSSH and for your feedback 🙏

You can already disable vibration inside WebSSH > Settings > SSH Settings > Enable bell vibration or beep

Have a nice day ☀️

**Comment by mscuttari** [comment]
Oh nice, I completely missed that option despite searching multiple times. Very thanks, have a nice day!

**Reply by isontheline** [reply]
You're Welcome @mscuttari
